MAINTAINERS.md

```diff
@@ -1,238 +1,82 @@
-# Maintainers' Guide
+# Maintaining
 
-## Branch organization
+## Overview
 
-The essence of this branching scheme is that you create "release branches" of
-independently releasable units of work.  These can then be released by their
-maintainers when ready.
+We use a three-stage release pipeline. Each stage of the release pipeline has
+its own dedicated branch and corresponding testing moon. Features and bug fixes
+progress through each stage--and are subject to testing along the way--until
+they're eventually released to the live network. This pipeline automates our
+release process, making it much easier to quickly and reliably ship code. It's
+also simple to reason about.
 
-### Master branch
+## Branches and Moons
 
-Master is what's released on the network.  Deployment instructions are in the
-next section, but tagged releases should always come from this branch.
+The branches and their corresponding moons that comprise the stages of the
+release pipeline are:
 
-### Feature branches
+--------------------------------------------------------------
+ Branch     |            Moon          | Target audience
+--------------------------------------------------------------
+`develop`   | `~binnec-dozzod-marzod`  | Kernel developers  
+`release`   | `~marnec-dozzod-marzod`  | Early Adopters  
```

Comment:
I thought we were gonna keep this secret to prevent people getting themselves into trouble?

Reply:
Yeah, this was important to Josh. I should put a loud warning here though.

WARNING: Using early pre-release moons (binnec and marnec) is high risk and may require you to breach if there's an issue.

Something like that.

Reply:
It's a calculated benefit. We have more qualified eyeballs to make bugs shallow than has been assumed to date.

WARNING: If you lack the requisite skills to troubleshoot and fix kernel issues, you should not sync from `develop/~binnec`. If you're not prepared to breach your ship in response to an issue stemming from an early release, do not use these pre-release moons.